backend/src/middleware/validation/index.js

```diff
@@ -22,10 +22,29 @@ const validateUrl = async (resolve, root, args, context, info) => {
   }
 }
 
+const validateComment = async (resolve, root, args, context, info) => {
+  const COMMENT_MIN_LENGTH = 1
+  const content = args.content.replace(/<(?:.|\n)*?>/gm, '').trim()
+  if (!args.content || content.length < COMMENT_MIN_LENGTH) {
+    throw new UserInputError(
+      `Comment must be at least ${COMMENT_MIN_LENGTH} character long!`
+    )
+  }
+  const NO_POST_ERR_MESSAGE = 'Comment cannot be created without a post!'
+  const { postId } = args
+  if (!postId) {
+    throw new UserInputError(NO_POST_ERR_MESSAGE)
+  }
+
+  return await resolve(root, args, context, info)
```

Comment:
@mattwr18 I am getting an ES-Linting error on line 39. error  Redundant use of `await` on a return value  no-return-await. Did you encounter this before? Any suggestions for a workaround?

Reply:
maybe you need to save the returned value from the `await resolve(root, args, context, info)` to a variable and return that?

Reply:
`return` automatically `awaits` therefore its redundant.
The Solution would be `to return resolve(...)` I guess

Reply:
@mattwr18 @ulfgebhardt Thanks for the input guys!

Reply:
@ALau2088 it seems that `return await resolve` is in fact different from `return resolve`. I guess, 99.9% of all cases it should make no difference and if it does, it's probably a code smell.

Nevertheless, I experiences issues when I removed the `await` [here](https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/middleware/validation/index.js#L7). So if you *want* disable the linter, you can do so by adding a comment (see the linked LOC above :point_up: ).